ckanext/cfpb_extrafields/plugin.py

```diff
@@ -275,6 +273,16 @@ def _modify_package_schema(self, schema):
                                       tk.get_converter('convert_to_extras'),],
             'privacy_has_direct_identifiers' : [tk.get_validator('ignore_missing'),
                                                 tk.get_converter('convert_to_extras'),],
+            'transfer_date' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
+            'data_governance_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
+            'legal_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
+            'pra:_notes' : [tk.get_validator('ignore_missing'),
+                                                tk.get_converter('convert_to_extras'),],
```

Comment:
Need to double check whether each of these 5 fields should have these 2 converters